src/main/java/org/jenkinsci/test/acceptance/junit/RuleAnnotation.java

```diff
@@ -30,12 +30,12 @@
 
     /**
      * Optional ordering among rules.
-     *
-     * Annotation with <code>priority >= 0</code> are guaranteed to be run after
+     * <p>
+     * Annotation with {@code priority >= 0} are guaranteed to be run after
      * Jenkins is up. Negative priorities are run before startup on best effort
      * basis. (It might not happen before for ExistingJenkinsController,
      * PooledJenkinsController and possibly others).
-     *
+     * <p>
```

Comment:
technically I think this is incorrect and should be `</p><p>`   but if the tool is happy then I think that is what we are aiming for.

Reply:
With javadoc 11 onwarads and its output using HTML 5 by default, we can use the appropriate shorthand to indicate a new paragraph. You don't necessarily need enclosing paragraph tags.